changelog.d/7801.doc

```diff
@@ -0,0 +1 @@
+Change documentation reg. apt-transport-https package, which is no longer needed on Debian 10/Ubuntu 18.04 and newer.
```

Comment:
```suggestion
Change the installation documentation regarding the `apt-transport-https` package, which is no longer needed on Debian 10/Ubuntu 18.04 and newer.
```

Also feel free to credit yourself by adding a "Contributed by..." mention at the end if you want :)

Reply:
Done.